task01/task01/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task01
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.ForegroundColor = ConsoleColor.Green;
+            Console.WriteLine("Молитва");
+            Console.ForegroundColor = ConsoleColor.Yellow;
```

Comment:
Название стихотворения лучше отделить пустой строкой
```suggestion
            Console.WriteLine("Молитва");
            
            Console.WriteLine();
            
            Console.ForegroundColor = ConsoleColor.Yellow;
```